### PR TITLE
Merge in arity check / opal master stuff

### PR DESCRIPTION
**Pull request by wied03**

### CODE REVIEW

File: lib/opal/rspec/rake_task.rb [hunk 1]
```diff
@@ -11,7 +11,19 @@ class RakeTask
       PORT = 9999
       URL = "http://localhost:#{PORT}/"
 
-      attr_accessor :pattern, :exclude_pattern, :files, :default_path, :runner, :timeout
+      attr_accessor :pattern, :exclude_pattern, :files, :default_path, :runner, :timeout, :arity_checking
+
+      def arity_checking?
+        current_opal = Gem::Dependency.new('opal', '>= 0.10').match?('opal', Gem::Version.new(Opal::VERSION).release.to_s)
```
Comment (elia) [inline lib/opal/rspec/rake_task.rb]: I think here you can do:

```rb
Gem::Version.new(Opal::VERSION) >= Gem::Version.new('0.10')
```

Comment (wied03) [inline lib/opal/rspec/rake_task.rb]: I don't think that picks up the pre-release versions properly

Comment (wied03) [inline lib/opal/rspec/rake_task.rb]:
```
rb(main):001:0> require 'opal'
=> true
irb(main):002:0> Opal::VERSION
=> "0.10.0.dev"
irb(main):003:0>  Gem::Version.new(Opal::VERSION) >= Gem::Version.new('0.10')
=> false
irb(main):004:0> Gem::Dependency.new('opal', '>= 0.10').match?('opal', Gem::Version.new(Opal::VERSION).release.to_s)
=> true
```

Comment (elia) [inline lib/opal/rspec/rake_task.rb]: solution in to use `0.10.0.a`:

```
⤑ irb                                                                          ~
irb(main):001:0> Gem::Version.new('0.10.0.dev') >= Gem::Version.new('0.10')
=> false
irb(main):002:0> Gem::Version.new('0.10.0.dev') >= Gem::Version.new('0.10.0.a')
=> true
```

**Pull request merged**